numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -241,6 +242,7 @@ def get_cuda_paths():
             'libdevice': _get_libdevice_paths(),
             'cudalib_dir': _get_cudalib_dir(),
             'static_cudalib_dir': _get_static_cudalib_dir(),
+            'include_dir': _get_include_dir(),
```

Comment:
The result of this should also be printed out in `numba.cuda.cudadrv.libs.test()`: https://github.com/NVIDIA/numba-cuda/blob/main/numba_cuda/numba/cuda/cudadrv/libs.py#L83

I know it's not libraries (and we might consider renaming that function / file) but the intent there is to debug all of Numba's CUDA file-searching functionality, and I think having the include paths also available in that debug output will be invaluable when people (inevitably, for this sort of thing) run into problems.

Reply:
This is added in https://github.com/NVIDIA/numba-cuda/pull/17/commits/b600fa027a3116bcb92f694461e96ac6fd442f31